App.js

```diff
@@ -231,15 +231,16 @@ function ChatScreen({route, navigation}) {
   );
 }
 
-const Stack = isSmallScreenWidth ? createNativeStackNavigator() : createWebNavigator();
+const Stack = createWebNavigator();
```

Comment:
I am not so sure about this change, I would much prefer to use the `createNativeStackNavigator` for small screens as that is tested and proven to work. This should work just fine with the styles breakpoint should it not?

Reply:
+1